FirebaseRemoteConfig/Sources/RCNConfigRealtime.m

```diff
@@ -103,6 +99,11 @@ @implementation RCNConfigRealtime {
   RCNConfigSettings *_settings;
   FIROptions *_options;
   NSString *_namespace;
+  NSInteger gMaxRetryCount;
```

Comment:
all the ivar should start with prefix '_'

Reply:
then you can remove the g if it's a iVar

Reply:
Updated